projects/templates/projects/view_project.html

```diff
@@ -149,6 +149,36 @@ <h4>Current Users</h4>
   {% endfor %}
 </ul>
 
+{% if is_pi %}
+  <h4>Pending Invitations</h4>
+
+  <form class="form-inline" role="form" method="post">
+    {% csrf_token %}
+    {% bootstrap_form invite_form %}
+    <div class="form-group">
+      <button type="submit" class="btn btn-default" name="invite_user">Invite user</button>
+    </div>
+  </form>
+
+  <ul class="list-unstyled">
+    {% for i in invitations %}
+    <li>
+      {{i.email_address}}
+      <form role="form" method="post" style="display:inline">
+        {% csrf_token %}
+        <input type="hidden" name="email_code" value="{{ i.email_code }}">
```

Comment:
If you're re-sending, it is safer to just include the ID of the invitation here instead of the code. The server can retrieve the code and re-send. No need to expose it in the HTML here, even in a hidden elem.

Reply:
Right now, clicking the resend icon generates a new invite object, with a new code, while expiring the previous invitation. I can still update this so that it's the invite ID here, but just want to clarify if this behavior is what we want.